doc/classes/EditorSceneImporterAssetImport.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<class name="EditorSceneImporterAssetImport" inherits="EditorSceneImporter" category="Core" version="3.1">
+	<brief_description>
+		This is a multi-format 3d asset importer.
+	</brief_description>
+	<description>
+		This is a multi-format 3d asset importer.
+
```

Comment:
Don't put empty lines in the doc XML, as each becomes a paragraph, so this would be an extra empty paragraph in the in-editor and online docs.

Reply:
Done.